Doc/whatsnew/3.8.rst

```diff
@@ -230,6 +230,21 @@ New Modules
 Improved Modules
 ================
 
+* The :meth:`_asdict()` method for :func:`collections.namedtuple` now returns
```

Comment:
These two items shouldn't be in this PR. Perhaps you need to rebase your branch again?